Gruntfile.js

```diff
@@ -24,15 +24,10 @@ module.exports = function(grunt) {
   // custom browserify transformer to re-write plugins to
   // self-register with Raven via addPlugin
   function AddPluginBrowserifyTransformer() {
+    var noop = function (chunk, _, cb) { cb(null, chunk); };
+    var append = function (cb) { cb(null, "\nrequire('../src/singleton').addPlugin(module.exports);"); };
     return function(file) {
-      return through(function(buf, enc, next) {
-        buf = buf.toString('utf8');
-        if (/plugins/.test(file)) {
-          buf += "\nrequire('../src/singleton').addPlugin(module.exports);";
```

Comment:
it's just luck that transform is getting called once with one buffer here.

Reply:
How so? It's completely valid `through2` syntax.

Reply:
so is `through(process.exit)` doesn't mean it's a good idea.

It's currently just luck that the current code works. If browserify presented you with a stream with multiple chunks you'd intersperse it with these require calls.

Reply:
Ah, I completely missed the point and forgot that buffer size is limited. Good catch, thanks